js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1074,6 +1075,8 @@ var EditingController = modules.ViewController.inherit((function() {
             if(deferreds.length) {
                 that._saving = true;
 
+                dataSource && dataSource._changeLoadingCount(1);
```

Comment:
Because DataSource does not have such public API. I can rename _changeLoadingCount method to changeLoadingCount or add beginLoading/endLoading methods to prevent using private API.